docker-compose.yml

```diff
@@ -223,6 +247,7 @@ networks:
 # Volumes used by the PEcAn stack. These volumes are used to make sure
 # we have persistent data. Uncomment driver_opts if you want to place
 # the data folders at a well known location.
+# can use sed -i~ 's/^    # /    /' docker-compose.yml
```

Comment:
Rather than uncommenting a version-controlled file, a better way to do this is via `docker-compose.override.yml`.

Reply:
I added a bit of information on this to `book_source/06_reference/04_docker/05_building_images.Rmd`. That might be a good place to add this documentation.

Reply:
done